pkg/network/java/testdata/docker-compose.yml

```diff
@@ -3,5 +3,9 @@ services:
   java:
     image: ${IMAGE_VERSION}
     entrypoint: java -cp /v ${ENTRYCLASS}
+    ulimits:
+      nofile:
+        soft: 1024
+        hard: 524288
```

Comment:
let's add a comment here as well explaining why we added it

Reply:
done